src/shm_sub.c

```diff
@@ -187,8 +187,8 @@ sr_shmsub_unlink(const char *name, const char *suffix1, int64_t suffix2)
         goto cleanup;
     }
 
-    /* unlink */
-    if (unlink(path) == -1) {
+    /* unlink and ignore non-fatal missing file as it may have been removed earlier or never created */
```

Comment:
Please also add the reason for that, just `(on a crash)` at the end, for example.